cfg_template.yml

```diff
@@ -1,17 +1,21 @@
 trains:
-  # This is an example. This PR train would have 4 branches plus 1 "combined" branch to run tests etc on.
-  #
-  # The order of branches is important - it defines what will be merged (or rebased) into what.
-  #
-  # The train identifier ("fix important things") is only there for readability and it can be any string. We
-  # suggest you use a short description explaining what the PR train is for.
-  #
-  # The "combined" branch is optional. If you include it, pr-train will create a "combined" branch for you.
-  #
-  # fix important things:
-  #   - zoe_branch_fix_first_thing
-  #   - zoe_branch_fix_the_other_thing
-  #   - zoe_branch_update_tests
-  #   - zoe_branch_regenerate_noisy_snapshots
-  #   - zoe_fix_the_important_things:
-  #       combined: true
+
+# This is an example. This PR train would have 4 branches plus 1 "combined" branch to run tests etc on.
+#
+# The order of branches is important - it defines what will be merged (or rebased) into what.
+#
+# The train identifier ("fix important things") is only there for readability and it can be any string. We
+# suggest you use a short description explaining what the PR train is for.
+#
+# The "combined" branch is optional. If you include it, pr-train will create a "combined" branch for you.
+#
+# This file should end up looking like:
+#
+# trains:
```

Comment:
Would this perhaps be confusing in that people would just uncomment this section and have `trains:` in the file twice?

Reply:
What confused me is that I thought the train identifier (`fix important things:`) was equivalent to the `trains:` line. But that's definitely user error on my part as the other docs show a complete example with both. And it's probably just me so go ahead and close this. :)

Reply:
Right, I hear what you're saying. I think it's probably safer to _not_ duplicate the `trains:` label as I can imagine that could easily cause errors & confusion.

So I'll close this - but I appreciate the PR and thanks for your effort! 🙇